menu/app.js

```diff
@@ -0,0 +1,146 @@
+  },
+  {
+    id: 3,
+    title: 'godzilla milkshake',
+    category: 'shakes',
+    price: 6.99,
+    img: './images/item-3.jpeg',
+    desc: 'ombucha chillwave fanny pack 3 wolf moon street art photo booth before they sold out organic viral.',
+  },
+  {
+    id: 4,
+    title: 'country delight',
+    category: 'breakfast',
+    price: 20.99,
+    img: './images/item-4.jpeg',
+    desc: 'Shabby chic keffiyeh neutra snackwave pork belly shoreditch. Prism austin mlkshk truffaut, ',
+  },
+  {
+    id: 5,
+    title: 'egg attack',
+    category: 'lunch',
+    price: 22.99,
+    img: './images/item-5.jpeg',
+    desc: "franzen vegan pabst bicycle rights kickstarter pinterest meditation farm-to-table 90's pop-up ",
+  },
+  {
+    id: 6,
+    title: 'oreo dream',
+    category: 'shakes',
+    price: 18.99,
+    img: './images/item-6.jpeg',
+    desc: 'Portland chicharrones ethical edison bulb, palo santo craft beer chia heirloom iPhone everyday',
+  },
+  {
+    id: 7,
+    title: 'bacon overflow',
+    category: 'breakfast',
+    price: 8.99,
+    img: './images/item-7.jpeg',
+    desc: 'carry jianbing normcore freegan. Viral single-origin coffee live-edge, pork belly cloud bread iceland put a bird ',
+  },
+  {
+    id: 8,
+    title: 'american classic',
+    category: 'lunch',
+    price: 12.99,
+    img: './images/item-8.jpeg',
+    desc: 'on it tumblr kickstarter thundercats migas everyday carry squid palo santo leggings. Food truck truffaut  ',
+  },
+  {
+    id: 9,
+    title: 'quarantine buddy',
+    category: 'shakes',
+    price: 16.99,
+    img: './images/item-9.jpeg',
+    desc: 'skateboard fam synth authentic semiotics. Live-edge lyft af, edison bulb yuccie crucifix microdosing.',
+  },
+  {
+    id: 10,
+    title: 'stake dinner',
+    category: 'dinner',
+    price: 32.99,
+    img: './images/item-10.jpeg',
+    desc: 'skateboard fam synth authentic semiotics. Live-edge lyft af, edison bulb yuccie crucifix microdosing.',
+  },
+];
+
+const sectionCenter = document.querySelector('.section-center');
+const btnContainer = document.querySelector('.btn-container');
+
+const displayMenuItems = (menuItem) => {
+  const displayMenu = menuItem
+    .map(
+      (item) => `
+   <article class="menu-item">
+        <img src=${item.img} alt=${item.title} class="photo" />
+        <div class="item-info">
+          <header>
+            <h4>${item.title}</h4>
+            <h4 class="price">$${item.price}</h4>
+          </header>
+          <p class="item-text">${item.desc}</p>
+        </div>
+      </article>
+  `
+    )
+    .join('');
+  sectionCenter.innerHTML = displayMenu;
+};
+
+const displayCategoryBts = () => {
+  const categories = menu.reduce(
+    (values, item) => {
+      if (!values.includes(item.category)) {
+        values.push(item.category);
+      }
+      return values;
+    },
+    ['all']
+  );
```

Comment:
Great work using reduce to get unique items in your array, however with ES6+ you could use `new Set()` to get unique values! 💭